modules/aioseop_sitemap.php

```diff
@@ -1777,6 +1885,20 @@ function get_sitemap_index_filenames() {
 				);
 			}
 
+			/**
+			 * Filter to update the indexed sitemap files.
+			 *
+			 * @since ?
+			 *
+			 * @param array $files {
+			 *      An array of arrays, where each array corresponds to the <url> element of the sitemap.
+			 *
+			 *      @type   array   The sitemap data corresponding to an <url> element. Each key should correspond to the name of
```

Comment:
The array had missing variable documentation.

```
/**
 * Filter to update the indexed sitemap files.
 *
 * @since ?
 *
 * @param array $files {
 *      An array of arrays, where each array corresponds to the <url> element of the sitemap.
 *
 *     @type array $index {
 *         The sitemap data corresponding to an <url> element. Each key should correspond to the name of
 *         the element as specified by the sitemap XSD.
 *
 *         @type string $changefreq Amount of time to update.
 *         @type string $loc        URL location of the Sitemap.
 *         @type string $priority   Set between "0.0" and "1.0".
 *     }
 * @param string $prefix The filename to prefix each sitemap name.
 * @param string $suffix The suffix/file extension to use.
 */
```